llvm/lib/Target/RISCV/RISCVISelLowering.cpp

```diff
@@ -5690,6 +5691,9 @@ static unsigned getRISCVVLOp(SDValue Op) {
   switch (Op.getOpcode()) {
   default:
     llvm_unreachable("don't have RISC-V specified VL op for this SDNode");
+  case ISD::VP_LRINT:
+  case ISD::VP_LLRINT:
+    return RISCVISD::VFCVT_X_F_VL;
```

Comment:
Minor nit, should we move this closer to the bottom with the other floating point VP ops